bin/rails

```diff
@@ -1,12 +1,5 @@
 #!/usr/bin/env ruby
-# Commented out for development purposes with Apple M1 Silicon Procesing Chips
-# UCLA Original: Keep -- Start
-# begin
-#   load File.expand_path('../spring', __FILE__)
-# rescue LoadError => e
-#   raise unless e.message.include?('spring')
-# end
-# UCLA Original: Keep --- End
+load File.expand_path("spring", __dir__)
```

Comment:
you brought spring back - is that ok?

Reply:
https://github.com/scientist-softserv/oral-history/pull/26/commits/fd543595260794a08a17d5dd3c9ed8b711330337

Reply:
@aprilrieger Let's do some more testing around this.